rm-datasource/src/main/java/io/seata/rm/datasource/exec/DeleteExecutor.java

```diff
@@ -61,10 +60,13 @@ protected TableRecords beforeImage() throws SQLException {
     }
 
     private String buildBeforeImageSQL(SQLDeleteRecognizer visitor, TableMeta tableMeta, ArrayList<List<Object>> paramAppenderList) {
-        String whereCondition = buildWhereCondition(visitor, paramAppenderList);
+        String orgSql = visitor.getOriginalSQL().replaceAll("(\\\r\\\n|\\\r|\\\n|\\\n\\\r)", " ");
```

Comment:
I think we don't need to check sql grammer.

Reply:
done